src/WithPagination.php

```diff
@@ -92,6 +103,6 @@ public function resolvePage()
     {
         // The "page" query string item should only be available
         // from within the original component mount run.
-        return (int) request()->query('page', $this->page);
+        return request()->query('page', $this->page);
```

Comment:
@ariaieboy please put the (int) cast back in.

Reply:
since cursor pagination not using integer for pagination instead it using encoded string casting to int will break the cursor pagination here

Reply:
Ah yep, no worries 🙂